certbot/main.py

```diff
@@ -554,6 +565,7 @@ def revoke(config, unused_plugins):  # TODO: coop with renewal config
 
     try:
         acme.revoke(jose.ComparableX509(cert), config.reason)
+        _want_delete(config)
```

Comment:
Please either rename this to a verb, like `_delete_if_appropriate`, or rewrite as `if _want_delete: cert_manager.delete()`.